target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4018-ex62x0v2.dtsi

```diff
@@ -0,0 +1,395 @@
+ * ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+ * OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+ *
+ */
+
+#include "qcom-ipq4019.dtsi"
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/soc/qcom,tcsr.h>
+
+/ {
+        model = "Netgear EX62x0v2";
+        compatible = "netgear,ex62x0v2";
+
+        soc {
+                rng@22000 {
+                        status = "okay";
+                };
+
+                mdio@90000 {
+                        status = "okay";
+                };
+
+                tcsr@1949000 {
+                        compatible = "qcom,tcsr";
+                        reg = <0x1949000 0x100>;
+                        qcom,wifi_glb_cfg = <TCSR_WIFI_GLB_CFG>;
+                };
+		ess_tcsr@1953000 {
+                        compatible = "qcom,tcsr";
+                        reg = <0x1953000 0x1000>;
+                        qcom,ess-interface-select = <TCSR_ESS_PSGMII>;
+                };
+
+                tcsr@1957000 {
+                        compatible = "qcom,tcsr";
+                        reg = <0x1957000 0x100>;
+                        qcom,wifi_noc_memtype_m0_m2 = <TCSR_WIFI_NOC_MEMTYPE_M0_M2>;
+                };
+
+                crypto@8e3a000 {
+                        status = "okay";
+                };
+
+                watchdog@b017000 {
+                        status = "okay";
+                };
+        };
+
+        aliases {
+                led-boot = &power_amber;
+                led-failsafe = &power_amber;
+                led-running = &power_green;
+                led-upgrade = &power_amber;
+                label-mac-device = &gmac;
+        };
+
+        keys {
+                compatible = "gpio-keys";
+
+                wps {
+                        label = "wps";
+                        gpios = <&tlmm 0 GPIO_ACTIVE_HIGH>;
+                        linux,code = <KEY_WPS_BUTTON>;
+                };
+
+                reset {
+                        label = "reset";
+                        gpios = <&tlmm 63 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+                };
+        };
+
+        led_spi {
+                compatible = "spi-gpio";
+                #address-cells = <1>;
+                #size-cells = <0>;
+
+                sck-gpios = <&tlmm 5 GPIO_ACTIVE_HIGH>;
+                mosi-gpios = <&tlmm 4 GPIO_ACTIVE_HIGH>;
+                num-chipselects = <0>;
+
+                led_gpio: led_gpio@0 {
+                        compatible = "fairchild,74hc595";
+                        reg = <0>;
+                        gpio-controller;
+                        #gpio-cells = <2>;
+                        registers-number = <1>;
+                        spi-max-frequency = <1000000>;
+                };
+        };
+
+        leds {
+                compatible = "gpio-leds";
+                power_amber: power_amber {
```

Comment:
Dont use underscores in node names, also the name should be `led-`, so:
`power_amber: led-power-amber`

The same applies for the rest as well